lib/taskana-core/src/test/java/acceptance/task/TransferTaskAccTest.java

```diff
@@ -241,22 +241,21 @@ void should_BulkTransferOnlyValidTasks_When_SomeTasksToTransferCauseExceptions()
     ArrayList<String> taskIdList = new ArrayList<>();
     taskIdList.add("TKI:000000000000000000000000000000000006"); // working
     taskIdList.add("TKI:000000000000000000000000000000000041"); // NotAuthorized READ
-    taskIdList.add("TKI:200000000000000000000000000000000006"); // NotAuthorized TRANSFER
+    taskIdList.add("TKI:200000000000000000000000000000000008"); // NotAuthorized TRANSFER
     taskIdList.add(""); // InvalidArgument
-    taskIdList.add(null); // InvalidArgument (added with ""), duplicate
-    taskIdList.add("TKI:000000000000000000000000000000000099"); // TaskNotFound
+    taskIdList.add(null); // InvalidArgument
+    taskIdList.add("TKI:000000000000000000000000000000000099"); // not existing
     taskIdList.add("TKI:100000000000000000000000000000000006"); // already completed
 
     BulkOperationResults<String, TaskanaException> results =
         taskService.transferTasks("WBI:100000000000000000000000000000000006", taskIdList);
     // check for exceptions in bulk
     assertThat(results.containsErrors()).isTrue();
-    assertThat(results.getErrorMap().values()).hasSize(5);
+    assertThat(results.getErrorMap().values()).hasSize(4);
```

Comment:
Why is there 1 error less now?

Reply:
Before there existed the method "removeNonExistingTasksFromTaskIdList" which put empty task Id's into the bulkLog (because of the property of a Map only once). I thought it would make more sense to not include NULL Ids as well as EMPTY Ids in our bulkLog. That's why the size of the bulkLog shrinked by one.